haystack/nodes/preprocessor/preprocessor.py

```diff
@@ -444,3 +450,49 @@ def _find_longest_common_ngram(
             # no common sequence found
             longest = ""
         return longest if longest.strip() else None
+
+    def _sentence_tokenizer(self, text: str) -> List[str]:
```

Comment:
Could we please rename that method to a verb-form emphasizing the action is being executed like`_split_sentences`? This would make it much clearer what to expect from that method. Currently it looks like passing some context (the text) to an object that at some later point might work on that context.